lib/screen/connection/persona_connections_page.dart

```diff
@@ -111,27 +120,98 @@ class _PersonaConnectionsPageState extends State<PersonaConnectionsPage>
     }
   }
 
+  Future<void>  _showConnectionOption() async {
+    final options = [
+      OptionItem(
+          title: "connect_via_clipboard".tr(),
+          icon: SvgPicture.asset("assets/images/DApp.svg"),
+          onTap: () async {
+            if (!mounted) return;
+            Navigator.of(context).pop();
+            try {
+              final clipboardData = await Clipboard.getData("text/plain");
+              if (clipboardData == null ||
+                  clipboardData.text == null ||
+                  clipboardData.text!.isEmpty) {
+                throw ConnectionViaClipboardError("Clipboard is empty");
+              }
+              final text = clipboardData.text!;
+              log.info("Connect via clipboard: $text");
+              _processDeeplink(text);
+            } catch (e) {
+              if (e is ConnectionViaClipboardError) {
+                if (!mounted) return;
+                UIHelper.hideInfoDialog(context);
+                UIHelper.showInvalidURI(context);
+              } else {
+                rethrow;
```

Comment:
log the error here instead of `rethrow`. rethrow will send the exception to the parent function then will be crashed if it didn't handle.